sockeye/log.py

```diff
@@ -128,6 +130,9 @@ def setup_main_logger(file_logging=True, console=True, path: Optional[str] = Non
     for _, handler_config in log_config['handlers'].items():  # type: ignore
         handler_config['level'] = level
 
+    if 'console' in log_config['handlers'] and console_level is not None:
```

Comment:
you might need a `# type: ignore` somewhere here for mypy to pass

Reply:
Yeah, I did that but hadn't noticed that my push was rejected because I needed to pull the master merge :)